torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9422,12 +9422,17 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                    ),
     UnaryUfuncInfo('conj',
                    ref=np.conj,
-                   dtypes=all_types_and_complex_and(torch.bool,
-                                                    torch.bfloat16, torch.half),
+                   dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16,
+                                                    torch.half, torch.complex32),
```

Comment:
you probably wanna enable this test for `conj_physical` which actually performs the conj computation